.github/workflows/release-binary.yml

```diff
@@ -0,0 +1,80 @@
+name: Release Binaries
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    name: Build and Upload Release Assets
+    runs-on: ubuntu-latest
+    container: golang:1.18-bullseye
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build Linux/amd64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=linux
+          export GOARCH=amd64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-linux-amd64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Windows/amd64 binary
+        run: |
+          apt update
+          apt -y install zip
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=windows
+          export GOARCH=amd64
+          go build -o cql-proxy.exe
+          zip -vr cql-proxy-windows-amd64-${{ github.ref_name }}.zip cql-proxy.exe LICENSE
+      - name: Build Darwin/amd64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=darwin
+          export GOARCH=amd64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-darwin-amd64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Darwin/arm64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=darwin
+          export GOARCH=arm64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-darwin-arm64-${{ github.ref_name }}.tgz cql-proxy LICENSE
+      - name: Build Linux/arm64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
+          export GOOS=linux
+          export GOARCH=arm64
+          go build -o cql-proxy
+          tar cvfz cql-proxy-linux-arm64-${{ github.ref_name }}.tgz cql-proxy LICENSE
```

Comment:
There's a lot of repetition of "go build" and "tar" commands here.  You should be able to define these operations once by using a strategy to specify which platforms/processors you wish to build for; the blocks in that strategy will then be applied for each operation.  Docs on strategies can be found [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow); there's also a decent [example](https://github.com/datastax/python-driver-wheels/blob/master/.github/workflows/build_wheels_mac.yml) in the python-driver-wheels repo.

Reply:
Fixed.